app/views/estimates/print.html.slim

```diff
@@ -44,7 +44,7 @@
     = render "evidence"
 
   - if @model.calculated?(:gross_income) || @model.calculated?(:disposable_income) || @model.calculated?(:capital)
-    h2.govuk-heading-m = t("estimates.show.#{@model.level_of_help}_how_calculated")
+    h2.govuk-heading-m.no-wrap = t("estimates.show.#{@model.level_of_help}_how_calculated")
```

Comment:
Because this class is still here, this text still falls off the screen in non-PDF print view on Chrome. I was hoping this wouldn't be needed any more.